### ISSUE TITLE
aria-setsize and aria-posinset not allowed on element "article" at this point

### ISSUE THREAD

**Issue by bcalou** [validator/validator#1322]
According to the [specs](https://w3c.github.io/aria/#aria-setsize), `aria-posinset`and `aria-setsize` can be used with the `article` tag.

Input : 
```
<!DOCTYPE html>
<html lang="">
<head>
<title>Test</title>
</head>
<body>
<div role="feed">
  <article aria-posinset="1" aria-setsize="1">
    <h2>My article</h2>
  </article>
</div>
</body>
</html>
```
Errors :
```
Attribute aria-posinset not allowed on element [article](https://html.spec.whatwg.org/multipage/#the-article-element) at this point
Attribute aria-setsize not allowed on element [article](https://html.spec.whatwg.org/multipage/#the-article-element) at this point.
```

Thanks for your work!

**Comment by scottaohara** [comment]
when child of a `role=feed`, yes, these attributes should be allowed.  otherwise, they do not make sense.  so more nuance needed for the rule here.

**Comment by bcalou** [comment]
Yes, you're right, that's a good point!

**Comment by dd8** [comment]
VNU is reporting 

> Error: Element article is missing required attribute role.

This looks wrong because `aria-posinset` and `aria-setsize` are allowed with `role=article` and the `<article>` element has an implicit role of article
